src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
I don't think we want to catch exceptions for that case do we? The old-style Deferred methods are supposed to throw immediately and not be caught.